### PR TITLE
Add documentation for new Identity Token `.well-known` endpoint used for Plugin WIF

### PR DESCRIPTION
**Pull request by vinay-gopalan**
Adds documentation for the endpoint added to Vault in #24898.

### CODE REVIEW

File: website/content/api-docs/secret/identity/tokens.mdx [hunk 1]
```diff
@@ -441,10 +441,43 @@ $ curl \
 
 ```json
 {
-  "issuer": "https://example.com:1234",
+  "issuer": "https://example.com/v1/identity/oidc",
```
Comment (austingebauer) [inline website/content/api-docs/secret/identity/tokens.mdx:444]: Thanks!

File: website/content/api-docs/secret/identity/tokens.mdx [hunk 2]
```diff
@@ -441,10 +441,43 @@ $ curl \
...
+$ curl \
+    --request GET \
+    http://127.0.0.1:8200/v1/identity/oidc/plugins/.well-known/openid-configuration
+```
+
+### Sample response
+
+```json
+{
+  "issuer": "https://example.com/v1/identity/oidc",
```
Comment (austingebauer) [inline website/content/api-docs/secret/identity/tokens.mdx:477]:
```suggestion
  "issuer": "https://example.com/v1/identity/oidc/plugins",
```

**Review by austingebauer — APPROVED** [review]
Thanks @vinay-gopalan

File: website/content/api-docs/secret/identity/tokens.mdx [hunk 3]
```diff
@@ -441,10 +441,43 @@ $ curl \
...
+  "id_token_signing_alg_values_supported": ["RS256"],
+  "scopes_supported": null,
+  "token_endpoint_auth_methods_supported": null,
+  "claims_supported": null
+}
+```
+
+## Read plugin identity .well-known configurations
+
+Query this path to retrieve a set of claims about the plugin identity tokens' configuration. The response is a compliant [OpenID Provider Configuration Response](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationResponse).
```
Comment (schavis) [inline website/content/api-docs/secret/identity/tokens.mdx:459]:
```suggestion
Use the token endpoint to retrieve an
[OpenID Provider Configuration Response](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationResponse)
with claims about the identity token configuration for a plugin.
```

Style correction: write in active voice

**Review by schavis — CHANGES_REQUESTED** [review]

File: website/content/api-docs/secret/identity/tokens.mdx (hunk 3 above)
Comment (vinay-gopalan) [reply website/content/api-docs/secret/identity/tokens.mdx]: Updated, thanks!

**Comment by vinay-gopalan** [comment]
@schavis I have made the changes that you had requested. Could you give the PR another review whenever you get the chance, thanks! 🙏🏼

**Review by schavis — APPROVED** [review]

**Pull request merged**